testcontainers-common/src/main/java/com/playtika/test/common/utils/ContainerUtils.java

```diff
@@ -46,21 +46,52 @@
 import java.util.Arrays;
 import java.util.function.Consumer;
 
+import static com.playtika.test.common.utils.DateUtils.NO_DATE_REPRODUCIBLE_BUILD;
+
 @Slf4j
 @UtilityClass
 public class ContainerUtils {
 
     public static final Duration DEFAULT_CONTAINER_WAIT_DURATION = Duration.ofSeconds(60);
 
+    /**
+     * Configure and start container (don't log build date, startup time and container messages)
+     */
+    public static GenericContainer<?> configureCommonsAndStart(GenericContainer<?> container,
+                                                               CommonContainerProperties properties) {
+        return configureCommonsAndStart(container, properties, null, null);
+    }
+
+    /**
+     * Configure and start container
+     * @param logger log build date and startup time, and container messages with default {@link #containerLogsConsumer(org.slf4j.Logger)} (don't log if {@code null})
+     */
     public static GenericContainer<?> configureCommonsAndStart(GenericContainer<?> container,
                                                                CommonContainerProperties properties,
                                                                Logger logger) {
+        return configureCommonsAndStart(container, properties, logger, null);
+    }
+
+    /**
+     * Configure and start container (don't log if {@code logger} and {@code logConsumer} are {@code null})
+     * @param logger log build date, startup time and container messages
+     * @param logConsumer log container messages with custom consumer (if {@code null} use {@code logger} with default {@link #containerLogsConsumer})
+     */
+    public static GenericContainer<?> configureCommonsAndStart(GenericContainer<?> container,
```

Comment:
I don't think this is a good idea to use any of the methods from `Utils` classes as a public API of the testcontainers-spring-boot project. @vasilievip what do you think?

Reply:
>I don't think this is a good idea to use any of the methods from

can you give a bit more details?